sdv/constraints/tabular.py

```diff
@@ -253,10 +290,11 @@ def _transform(self, table_data):
                 Transformed data.
         """
         table_data = table_data.copy()
-        low_column = table_data[self._low]
-        diff = table_data[self._high] - low_column
+        low = self._low if self._low_is_scalar else table_data[self._low]
```

Comment:
nit: Could pull this into a class methods `get_low_value`, `get_high_value` since this logic is repeated so many times